### ISSUE TITLE
TeardownClass fixture run multiple times

### ISSUE THREAD

**Issue by mrvkino** [kondratyev-nv/vscode-python-test-adapter#73]
Hi,

I am testing file access in my code and I have a Unittest written like this.

```python
class TestClass(unittest.TestCase):

    @classmethod
    def setUpClass(cls):
        # create files
        return super().setUpClass()
    
    def test_files_access_1(self):
        # test files access
    
    def test_files_access_2(self):
        # test files access
    
    @classmethod
    def tearDownClass(cls):
        # delete files
        return super().tearDownClass()
```

If I run the test with `python -m unittest discover` all my tests passes. If I run *all* the tests from the test explorer sidebar, "test_file_access_2" will fail because the files are deleted. But, everything works if I run my test from the test file view.

![image](https://user-images.githubusercontent.com/279511/55438107-b43a6480-556e-11e9-989f-8c9a9a82108a.png)

This does not happen if I run each test *one by one*.  I can make my test pass, when I do a run all, if I comment the tearDownClass method.

It seems the tearDownClass is run multiple time (once per method test to run?). Could it be possible to have the same behaviour than Unittest discover (tearDownClass run after all test from class are finished)?

**Comment by kondratyev-nv** [comment]
@mrvkino Thank you for the issue! Indeed, this is a bug, and I will try to fix it.

**Comment by kondratyev-nv** [comment]
@mrvkino I have published release 0.3.4. Please, update the extension and let me know if the issue persists.

**Comment by mrvkino** [comment]
It has resolved the issue. Thanks!
